docs/creating-service-bindings/custom-path-injection.md

```diff
@@ -6,9 +6,12 @@ sidebar_position: 3
 
```

Comment:
We should have an intro that explain what that is about, the different options, when to use them.